tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenAssertCollectionNotExists.cs

```diff
@@ -0,0 +1,76 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using MongoDB.Bson;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public class JsonDrivenAssertCollectionNotExists : JsonDrivenTestRunnerTest
```

Comment:
Note to other reviewers: I found this name to be odd too but it's the name the spec has:
https://github.com/mongodb/specifications/commit/de5b560ebe2ba28d9569031ed8ea1df3164eeef7#diff-187553190827e5ea79f632ef895251a1R403